README.md

```diff
@@ -34,7 +34,7 @@ You'll need [Docker](https://www.docker.com/get-docker), [Docker Compose](https:
 
 1. cd .../mysql-backup-restore
 2. mkdir ./mybackups
-3. cp -p test/world.sql.gz ./mybackups
+3. Upload test/world.sql.gz to the S3 bucket.
```

Comment:
Can you help me understand why this was changed?

Reply:
I don't know.

Reply:
I did it.  The test sequence needs the world.sql.gz file populated in the S3 bucket because the first operation is to *restore* the DB from the backup in the S3 bucket.  I also need to remove the 'mkdir ./mybackups' since the directory is no longer needed.  I believe these were holdovers from a previous example usage.